src/components/ModelTableList/CloudGroup.js

```diff
@@ -67,48 +63,57 @@ export default function CloudGroup({ activeUser, filters }) {
       modelGroup.forEach((model) => {
         const { highestStatus } = getModelStatusGroupData(model);
         cloudModels.rows.push({
+          "data-test-model-id": model?.uuid,
```

Comment:
Maybe we should be more specific in our `data-test` attributes, so 'id' to 'uuid' here? If you agree the other files will need to be updated as well.

```suggestion
          "data-test-model-uuid": model?.uuid,
```

Reply:
Good idea.